modules/loki-network-policies.adoc

```diff
@@ -0,0 +1,23 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-10-17
+:_mod-docs-content-type: CONCEPT
+
+[id="loki-network-policies_{context}"]
+= Loki network policies
+
+You can enable the {loki-op} to automatically create a `NetworkPolicy` resource that implements a "default deny" security model with explicit allow rules for required communications. 
+Network policies provide network segmentation for your LokiStack deployment by controlling ingress and egress traffic between Loki components and external services.
+The network policies in {loki-op} are designed to be secure by default while maintaining compatibility across diverse environments.
+
+Network policies for Loki on {ocp-product-title} include the following additional integrations:
+
+* Monitoring: Automatic integration with the {ocp-product-title} monitoring stack.
+* DNS: Support for both standard and {ocp-product-title} DNS services (port 5353).
+AlertManager: Built-in access to cluster monitoring AlertManager.
```

Comment:
It's already part of the monitoring stack
```suggestion
```

Reply:
done